rtlib/include/rt/Data.hpp

```diff
@@ -0,0 +1,71 @@
+//
+// Created by Anthony Tamasi on 2019-10-17.
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include <boost/filesystem.hpp>
+#include <opencv2/core.hpp>
+
+#include "rt/ImageTypes.hpp"
+#include "rt/types/ITKMesh.hpp"
+
+namespace et = envitools;
+namespace fs = boost::filesystem;
```

Comment:
Don't use namespace aliases in class headers.

Reply:
How do you declare that these will be used in this case? Would it be as simple as
```
namespace envitools;
```
or is it declared another way?

Reply:
Or can you do 
```
using namespace envitools;
using namespace boost::filesystem;
```

using several namespaces in the same file?